### PR TITLE
[ Xedra Evolved ] March Lord Banners

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Adds banners that G.U.T requested in discord to the changeling bannermen.

#### Describe the solution
Adds banner items and spawns

#### Describe alternatives you've considered
I'm open to suggestiosn and would appreciate additional banners with more exotic abilities and descriptions.  Blue orange morality type stuff welcome.

### CODE REVIEW

**Comment by Rokharn** [comment]
banner of fae spirit
A back mounted flag identifying the loyalties of troops. The heraldry on this banner is of the Great Tree around which framed bizarre creatures dance. You feel like you are part of something incredible.
"MAX_MANA" +200 (maybe multiply 1.05-1.1 will be better)

banner of blinding courage 
A back mounted flag identifying the loyalties of troops. The heraldry on this banner is of the huge bear furiously fighting with a whole cloud of bees.  
"ATTACK_SPEED" +10 , "ATTACK_NOISE" multiply 1.5, vision range -3 ("MOTION_VISION_RANGE"?)

File: data/mods/Xedra_Evolved/items/armor/changeling.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    "material_thickness": 2,
+    "flags": [ "BELTED" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 1 } ] } ] },
+    "armor": [ { "encumbrance": [ 2, 25 ], "coverage": 60, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
+  },
+  {
+    "id": "banner_starry_wisdom",
+    "type": "ARMOR",
+    "name": { "str": "banner of starry wisdom", "str_pl": "banners of starry wisdom" },
+    "description": "A back mounted flag identifying the loyalties of troops. The heraldry on this banner is of the night sky shining with ancient wisdom, full of stars.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/armor/changeling.json:63]:
```suggestion
    "description": "A back mounted flag identifying the loyalties of troops.  The heraldry on this banner is of the night sky shining with ancient wisdom, full of stars.",
```

File: data/mods/Xedra_Evolved/items/armor/changeling.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    "type": "enchantment",
+    "id": "thunder_rush",
+    "condition": "ACTIVE",
+    "values": [ { "value": "FOOTSTEP_NOISE", "multiply": 1.5 }, { "value": "SPEED", "add": 10 } ]
+  },
+  {
+    "id": "banner_thunderous_rush",
+    "type": "ARMOR",
+    "name": { "str": "banner of thunderous rush", "str_pl": "banners of starry wisdom" },
+    "description": "A back mounted flag identifying the loyalties of troops. The heraldry on this banner is of lightning striking a tower that is falling.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/armor/changeling.json:88]:
```suggestion
    "description": "A back mounted flag identifying the loyalties of troops.  The heraldry on this banner is of lightning striking a tower that is falling.",
```

File: data/mods/Xedra_Evolved/items/armor/changeling.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  {
+    "type": "enchantment",
+    "id": "thunder_rush",
+    "condition": "ACTIVE",
+    "values": [ { "value": "FOOTSTEP_NOISE", "multiply": 1.5 }, { "value": "SPEED", "add": 10 } ]
+  },
+  {
+    "id": "banner_thunderous_rush",
+    "type": "ARMOR",
+    "name": { "str": "banner of thunderous rush", "str_pl": "banners of starry wisdom" },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/armor/changeling.json:87]:
```suggestion
    "name": { "str": "banner of thunderous rush", "str_pl": "banners of thunderous rush" },
```

**Comment by Rokharn** [comment]
"banner_group" contain 3 of 5 banners. Was it intended that way or just missed id?

File: data/mods/Xedra_Evolved/itemgroups/itemgroups.json [hunk 4]
```diff
@@ -219,13 +219,24 @@
...
     ]
   },
+  {
+    "type": "item_group",
+    "id": "banner_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "banner_dark_forest", "prob": 20 },
+      { "item": "banner_butterfly_knight", "prob": 20 },
+      { "item": "banner_angry_darkness", "prob": 20 }
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/itemgroups/itemgroups.json:237]:
```suggestion
      { "item": "banner_angry_darkness", "prob": 20 },
      { "item": "banner_starry_wisdom", "prob": 20 },
      { "item": "banner_thunderous_rush", "prob": 20 }
```

**Comment by Maleclypse** [comment]
> "banner_group" contain 3 of 5 banners. Was it intended that way or just missed id?

Missed.  If you want to PR some more once this is merged go for it.  I've got other PRs I need to get finished.  Also I'm really hoping for more alien/conditional effects than I've currently got for banners.  Things that check if you're in the forest or in a city to determine if they give effects or not, or one that checks for a trait, things that are more complicated than here's a bonus.

**Comment by Rokharn** [comment]
>  Things that check if you're in the forest or in a city to determine if they give effects or not

Something like fae bans (in city, in forest, etc), but on relics? This is out my skills right now, but I have idea, how to 'cook' it. Relics can add temporally mutations, mutations can be using for eoc checks, similar to eoc "EOC_ARVORE_FAE_BAN_SLEPT_IN_CITY" (or other of this kind, made by Standing-Storm) but for other purposes.

```
"relic_data": { "passive_effects": [ { "id": "example_id" } ] },
  {
    "id": "ench_id",
    "type": "enchantment",
    "has": "WORN",
    "condition": "ALWAYS",
    "mutations": [
      "example_mutation",
    ]
  },
```

sleep in city eocs can be good base for new ones
 https://github.com/CleverRaven/Cataclysm-DDA/pull/70979/files#diff-2f648792e67159edba807bb64ef7fb537886c2eccfb5efcdb3c3078ce9f2b7e0

Possible to add positive effects via eocs too.

**Pull request merged**